client/my-sites/stats/stats-module-utm-builder/stats-module-utm-builder.tsx

```diff
@@ -26,14 +27,22 @@ const UTMBuilder: React.FC< Props > = ( { modalClassName } ) => {
 
 	return (
 		<>
-			<Button
-				icon={ link }
-				className="stats-utm-builder__trigger"
-				onClick={ handleClick }
-				variant="secondary"
-			>
-				{ translate( 'URL Builder' ) }
-			</Button>
+			{ /* div wrapper is needed to support custom trigger elements with an interal click event */ }
+			{ trigger ? (
+				// eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/no-noninteractive-tabindex
+				<div onClick={ handleClick } onKeyDown={ handleClick } tabIndex={ 0 }>
```

Comment:
I'm on the fence if this is a good way around it or cloning an element and passing an internal onClick would be better. Now I'm leaning toward the other way.

Reply:
I replaced this.